apps/src/templates/progress/progressHelpers.js

```diff
@@ -70,11 +70,7 @@ export function stageLocked(levels) {
   // an identical locked/unlocked state.
   // Given this, we should be able to look at the last level in our collection
   // to determine whether the LG (and thus the stage) should be considered locked.
-  const level = levels[levels.length - 1];
-  return (
-    level.status === LevelStatus.locked ||
-    (level.kind === 'assessment' && level.status === 'submitted')
-  );
+  return levels[levels.length - 1].locked;
```

Comment:
`locked` will be undefined until the async fetch completes and will stay undefined if we don't have progress for the level, so this needs to be `return !!levels[levels.length - 1].locked;` to avoid prop validation errors because we pass the return value of this to a `PropTypes.bool.isRequired`